scripts/rollup/config.js

```diff
@@ -30,3 +35,23 @@ export const plugins = [
     showGzippedSize: true,
   }),
 ];
+const external = isReactBuild ? ['react', 'react-dom'] : undefined;
+const globals = isReactBuild
+  ? { react: 'React', 'react-dom': 'ReactDOM' }
+  : undefined;
```

Comment:
could this not be an argument to createRollupConfig?

Reply:
I think an argument makes more sense, since you don't want to toggle between react and non-react builds for the same package, but either way works indeed

Reply:
I use another function called `createRollupConfigForReact` so we don't depend on env variables in 2c5389228058fe014449b3b963eb940539f9bbce.